lib/modules/datasource/aws-machine-image/readme.md

```diff
@@ -1,4 +1,7 @@
-> :warning: **This datasource is _experimental_**: Be aware that its syntax and behavior may change at any time!
+<!-- prettier-ignore -->
```

Comment:
Do we publish this readme on our docs site? I cannot find it when doing a search on the production site.

Reply:
Do we copy other readmes into the published docs? Or is it just this one that we forgot in a build-script somewhere?

Reply:
We don't read those files yet. We should do this for datasources / versionings and platforms, like we do for managers.

Reply:
I found issue #8855, should we add the "copy/paste `readme.md` into published docs" step in there? Or do you want a completely new issue to track the idea?

Reply:
I think new issue, it can be solved independently

Reply:
I made issue #14608 to track the work needed to publish the `readme.md` files. 😉